src/aws_lambda_typing/events/s3_batch.py

```diff
@@ -73,13 +79,13 @@ class S3BatchResponseResult(TypedDict):
     ----------
     taskId: str
 
-    resultCode: str
+    resultCode: S3BatchResponseResultCode
```

Comment:
When usin `resultCode: S3BatchResponseResultCode` the docstring doesn't display correctly, can you instead use `Literal['Succeeded', 'TemporaryFailure', 'PermanentFailure']` in all four docstrings?

![image](https://user-images.githubusercontent.com/16031346/149411085-92ca65c9-fb9c-4a3d-b248-7374865328ca.png)

![image](https://user-images.githubusercontent.com/16031346/149411208-9e3bbcc1-3cc0-4e78-9255-a23a1262bb53.png)